swift-ci/sdks/android/scripts/build.sh

```diff
@@ -0,0 +1,704 @@
+        rm -f $ndk_installation/sysroot/usr/lib/swift
+    quiet_popd
+    groupend
+done
+
+# Now generate the bundle
+groupstart "Bundling SDK"
+
+sdk_base=swift-android
+sdk_staging="sdk_staging"
+
+bundle="${sdk_name}.artifactbundle"
+
+rm -rf ${build_dir}/$bundle
+mkdir -p ${build_dir}/$bundle
+quiet_pushd ${build_dir}/$bundle
+
+# First the info.json, for SwiftPM
+cat > info.json <<EOF
+{
+  "schemaVersion": "1.0",
+  "artifacts": {
+    "$sdk_name": {
+      "variants": [
+        {
+          "path": "$sdk_base"
+        }
+      ],
+      "version": "${android_sdk_version}",
+      "type": "swiftSDK"
+    }
+  }
+}
+EOF
+
+mkdir -p $sdk_base
+quiet_pushd $sdk_base
+
+cp -a ${build_dir}/sdk_root ${sdk_staging}
+
+swift_res_root="swift-resources"
+mkdir -p ${swift_res_root}
+
+cat > $swift_res_root/SDKSettings.json <<EOF
+{
+  "DisplayName": "Swift Android SDK",
+  "Version": "${android_sdk_version}",
+  "VersionMap": {},
+  "CanonicalName": "linux-android"
+}
+EOF
+
+# Copy necessary headers and libraries from the toolchain and NDK clang resource directories
+mkdir -p $swift_res_root/usr/lib/swift/clang/lib
+cp -r $host_toolchain/lib/clang/*/include $swift_res_root/usr/lib/swift/clang
+
+for arch in $archs; do
+    quiet_pushd ${sdk_staging}/${arch}/usr
+        rm -r bin
+        rm -r include/*
+        cp -r ${swift_source_dir}/swift/lib/ClangImporter/SwiftBridging/{module.modulemap,swift} include/
+
+        arch_triple="$arch-linux-android"
+        if [[ $arch == 'armv7' ]]; then
+            arch_triple="arm-linux-androideabi"
+        fi
+
+        # need force rm in case linux is not present (when not running tests)
+        rm -rf lib/swift{,_static}/{FrameworkABIBaseline,_InternalSwiftScan,_InternalSwiftStaticMirror,clang,embedded,host,linux,migrator}
+        rm -rf lib/lib*.so
+        mv lib/swift lib/swift-$arch
+        ln -s ../swift/clang lib/swift-$arch/clang
+
+        mv lib/swift_static lib/swift_static-$arch
+        mv lib/lib*.a lib/swift_static-$arch/android
+
+        ln -sv ../swift/clang lib/swift_static-$arch/clang
+
+        # copy the clang libraries that we need to build for each architecture
+        aarch=${arch/armv7/arm}
+        mkdir -p lib/swift/clang/lib/linux/${aarch}
+
+        # match clang version 21, 22, etc.
+        cp -av ${ndk_installation}/lib/clang/[0-9]*/lib/linux/libclang_rt.builtins-${aarch}-android.a lib/swift/clang/lib/linux/
+        cp -av ${ndk_installation}/lib/clang/[0-9]*/lib/linux/${aarch}/libunwind.a lib/swift/clang/lib/linux/${aarch}/
+    quiet_popd
+
+    # now sync the massaged sdk_root into the swift_res_root
+    rsync -a ${sdk_staging}/${arch}/usr ${swift_res_root}
+done
+
+rm -r ${swift_res_root}/usr/share/{doc,man}
+rm -r ${sdk_staging}
+
+# create an install script to set up the NDK links
+#ANDROID_NDK_HOME="/opt/homebrew/share/android-ndk"
+mkdir scripts/
+
+ndk_sysroot="ndk-sysroot"
+
+cat > scripts/setup-android-sdk.sh <<'EOF'
+#/bin/bash
```

Comment:
Should this be a shebang?

```suggestion
#!/bin/env bash
```

Reply:
I've switched it to be #!/usr/bin/env bash